tests/verifyPMMInventory_test.js

```diff
@@ -2,7 +2,10 @@ const assert = require('assert');
 
 Feature('Inventory page');
 
-Before(async ({ I }) => {
+let pmmVersion;
+
+Before(async ({ I, homePage }) => {
+  pmmVersion = await homePage.getVersions().versionMinor;
```

Comment:
I don't think we need to fetch the version here. The test intended for dev-latest

Reply:
removed pmm version from tests.